README.md

```diff
@@ -20,15 +20,25 @@ If there are any issues with these instructions or if anything is unclear, pleas
 
 4. Add secret `.env` file to `App` dir. Ask in Slack for the file. Add `.` to the front of the file in the directory so it is invisible. (Filename should be: `.env`) **This should never be pushed to GitHub!**
 
-5. [Install PostgreSQL](https://www.postgresql.org/download/)
+5. Set up the database:
+    - [Install PostgreSQL](https://www.postgresql.org/download/)
+    - (Windows) Set environment variables after install following [step 3 of this guide](https://medium.com/@aeadedoyin/getting-started-with-postgresql-on-windows-201906131300-ee75f066df78)
+    - Start server and start psql
+
+        ```bash
+        pg_ctl start
+        psql postgres
```

Comment:
I installed PostgreSQL with Homebrew so the server starts automatically for me and i only need to run`psql postgres` to start psql - not sure if you want to include this as a note in the README

Reply:
Added!